src/Integration/Service/SonarQubeServiceWrapper.cs

```diff
@@ -40,10 +42,13 @@ internal class SonarQubeServiceWrapper : ISonarQubeServiceWrapper
         public const string ProjectDashboardRelativeUrl = "dashboard/index/{0}";
 
         public const string RoslynExporter = "roslyn-cs";
-
-        public const string CSharpLanguage = "cs";
-        public const string VBLanguage = "vbnet";
-        public static readonly string[] SupportedLanguages = { CSharpLanguage, VBLanguage };
+        
+        private static readonly IReadOnlyDictionary<Language, string> languageKeys = new ReadOnlyDictionary<Language, string>(
```

Comment:
The old `Language.ServerKey` was removed. This mapping is local to the `SonarQubeServiceWrapper` since the specific string key is a server specific detail which should be hidden from the consumers of the service (as this might not be stable between plugin versions etc.).